hardhat.config.ts

```diff
@@ -125,7 +125,7 @@ const config: HardhatUserConfig = {
       verifyURL: "https://zksync2-mainnet-explorer.zksync.io/contract_verification",
     },
     optimism: {
-      url: getNodeUrl("optimism", true, 10),
+      url: getNodeUrl("optimism-mainnet", true, 10),
```

Comment:
Is this right? How has this worked before?

Reply:
Yeah - the first variable is the subdomain. On infura it's `optimism-mainnet` not `optimism`